src/mots/cli.py

```diff
@@ -235,7 +260,10 @@ def create_parser():
         default=DEFAULT_CONFIG_FILEPATH,
     )
     export_parser.add_argument(
-        "--format", "-f", type=str, default="rst", help="the format of exported data"
+        "--format", "-f", type=str, help="the format of exported data"
+    )
+    export_parser.add_argument(
+        "--out", "-o", type=str, help="the file path to output to"
```

Comment:
You can use `type=pathlib.Path` to keep paths consistent, and then use the `pathlib` opener APIs instead of `open()` elsewhere.

Reply:
Noted.